js/src/toast.js

```diff
@@ -64,6 +68,8 @@ class Toast extends BaseComponent {
 
     this._config = this._getConfig(config)
     this._timeout = null
+    this._activeMouseInteraction = false
+    this._activeKeyboardInteraction = false
```

Comment:
please, try to follow boolean variables naming convention, putting the prefix 'has'